packages/Panel/stories/examples/BasicPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Button from "@paprika/button";
+import Panel from "../../src";
+import { Nav, TextLine } from "../helpers";
+
+export default function BasicPanel(props) {
+  const { disableBodyOverflow, hasOverlay, slideFrom } = props;
+  const [isOpen, setIsOpen] = React.useState(true);
+  const toggle = () => {
+    setIsOpen(state => !state);
+  };
+
+  return (
+    <React.Fragment>
+      <Nav />
+      <Panel
+        a11yText="Panel View"
+        disableBodyOverflow={disableBodyOverflow}
+        isOpen={isOpen}
+        onClose={toggle}
+        offset={{ top: 40 }}
```

Comment:
For the "Basic" example, maybe it can open _over_ the `<Nav>` and this can be extra simple?